<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Video</class>
 <widget class="QWidget" name="Video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="video" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#video{
	border-image: url(:/imgs/video_bg.jpg);
}
QPushButton#on_stopBtn[mode='pause']{
   border-image: url(:/imgs/btn_play_normal.png);
}
QPushButton#on_stopBtn[mode='pause']:hover{  
   border-image: url(:/imgs/btn_play_hover.png)
}

QPushButton#on_stopBtn[mode='play']{
   border-image: url(:/imgs/btn_pause_normal.png);
}
QPushButton#on_stopBtn[mode='play']:hover{
   border-image: url(:/imgs/btn_pause_hover.png)
}
QPushButton#preBtn{
  
	border-image: url(:/imgs/btn_previous_normal.png);
	
}
QPushButton#nextBtn{
  
	border-image: url(:/imgs/btn_next_normal.png);
}
QPushButton#preBtn:hover{
  
	border-image: url(:/imgs/btn_previous_hover.png);
}
QPushButton#nextBtn:hover{
  
	border-image: url(:/imgs/btn_next_hover.png);
}</string>
   </property>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#backBtn{
	border-image: url(:/imgs/back.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="voiceBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>535</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgs/voiceopen.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>本地视频</string>
     </attribute>
     <widget class="QListWidget" name="localList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>205</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>在线视频</string>
     </attribute>
     <widget class="QListWidget" name="onlineList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>205</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="videoslider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="totalLab">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="on_stopBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="voiceslider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openVideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgs/btn_local_file.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="voiceLab">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bigBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgs/screen.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>551</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
